index.js

```diff
@@ -1,5 +1,40 @@
 function scoreCalculator(arr) {
   // do work here
+  let points;
+  let pointsArr = [];
+
+  let bonus = arr.every(e => {
+    return e < 5;
+  });
+  if (bonus === true) {
+    arr.map(e => {
+      points = 10;
+      pointsArr.push(points);
+    });
+    pointsArr.push(100);
+  } else {
+    arr.filter(e => {
+      if (e === []) {
+        points = -1;
```

Comment:
You should just return `-1` from here since points isn't really doing anything at this line